jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/JbpmBpmn2TestCase.java

```diff
@@ -144,7 +144,7 @@
     private RequirePersistence testReqPersistence;
 
     @Rule
-    public Timeout globalTimeout = Timeout.seconds(30);
+    public Timeout globalTimeout = Timeout.seconds(300);
```

Comment:
Is this really necessary? Cannot we override the rule with higher timeout in the test class where it is really needed? Waiting 5 minutes for every single test is quite a lot of time and if something goes wrong with more tests, it will take a while to finish all the tests in this module.